core/src/main/res/values/strings.xml

```diff
@@ -197,6 +205,8 @@
     <string name="remove_new_flag_label">Remove \"new\" flag</string>
     <string name="removed_new_flag_label">Removed \"new\" flag</string>
     <string name="mark_read_label">Mark as played</string>
+    <string name="marked_read_label">Marked as played</string>
+    <string name="marked_unread_label">Marked as unplayed</string>
```

Comment:
Please use the name `marked_unplayed_label` here. I know the one above still uses a legacy name but for new strings we can use the up-to-date meaning.

Reply:
did/could you fix this?